### ISSUE TITLE
Animation effects are used by the UI despite Windows Accessibility settings

### ISSUE THREAD

**Issue by tksuoran** [files-community/Files#13228]
### Description

I have disabled animation effects in Windows Settings / Accessibility / Visual effects. Despite this, files app is using smooth scrolling.

### Steps To Reproduce

1. In Windows Settings / Accessibility / Visual effects, turn Animation effects off
2. Observe Files app UI using smooth scrolling for example

### Requirements

When Animation effects is turned off in Windows Settings, Files app UI disables animation effects.

### Files Version

2.5.21.0

### Windows Version

10.0.22621.2134

### Log File

[debug.log](https://github.com/files-community/Files/files/12397915/debug.log)

**Comment by 0x5bfa** [comment]
I'm not sure. but I believe so as well. @chingucoding would know?

**Comment by chingucoding** [comment]
I'm fairly sure that ScrollViewer does not have an option to turn this off. It does respect the setting but just not for smooth scrolling as far as I can tell.

**Comment by yaira2** [comment]
@tksuoran thank you for your feedback, it appears that this is a limitation in the WinAppSdk framework. We suggest submitting this feedback on the WinAppSdk or WinUI repos on GitHub.

**Comment by panbeep** [comment]
Is it possible to disable UI animations if Files app?

**Comment by Josh65-2201** [comment]
You can turn them off globally in Windows. There isn't a settings just for Files as there's no reason to turn off animations.

**Comment by XP1** [comment]
> There isn't a settings just for Files as there's no reason to turn off animations.

The worst animation is the smooth scrolling (which is the main point of the issue). I want scrolling to be instant, not sluggishly updating the screen and making me dizzy.

I tried to disable ScrollViewer scroll animation using `<ScrollViewer IsScrollInertiaEnabled="False">`, but the scrolling is extremely jittery. I have filed a bug report for [microsoft-ui-xaml](https://github.com/microsoft/microsoft-ui-xaml) here:

Scrolling is terrible when ScrollViewer IsScrollInertiaEnabled is False:
https://github.com/microsoft/microsoft-ui-xaml/issues/9124
